ludwig/data/dataset/ray.py

```diff
@@ -60,20 +60,15 @@ def read_remote_parquet(path: str):
 
 @DeveloperAPI
 class RayDataset(Dataset):
-    """Wrapper around ray.data.Dataset.
-
-    Attributes:
-        auto_window: If True and the dataset is larger than available memory,
-            automatically set window size to `<available memory> // 5`.
-    """
+    """Wrapper around ray.data.Dataset."""
```

Comment:
nit: While we're updating the doc strings, should we update it to all of the inputs and their types for this class? toally ok to skip as well

Reply:
Ah, good catch. I meant to add `window_size_bytes` back in and fill in the rest of the docstring.

Reply:
Added in [db4218b](https://github.com/ludwig-ai/ludwig/pull/3076/commits/db4218bbb5404fb45e3a7795b6177eea0f05484e)